### DISCUSSION TITLE
Just getting started (error)

### DISCUSSION THREAD

**Discussion by adivated** [prisma/prisma#4317] [Q&A] [answered]
using postgresql in docker container localhost
prisma init, introspect and generate all worked, etc.

just trying to run app gives:

Error: Error: 
    at PrismaClientKnownRequestError2 (file:///Users/rr/Projects/ew/node_modules/@prisma/client/runtime/index.js:4215:6)
    at request (file:///Users/rr/Projects/ew/node_modules/@prisma/client/runtime/index.js:78782:14)
    at processTicksAndRejections (node:internal/process/task_queues:93:4)

any help appreciated. Not overly helpful message 🤷‍♂️

"@prisma/client": "^2.11.0"
node v15.2.0

**Comment by ryands17** [comment, marked as answer]
Hey @Rjrunner44 👋 
Could you try with Node version 12 or 14 and see if it works?

If it still doesn't work, I would suggest sending me a reproduction repo of your code so that I can check.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
